guides/get-started-microservices-on-kubernetes/simple-microservice-part1.adoc

```diff
@@ -0,0 +1,267 @@
+...
+16:14:50,403 INFO  [org.jboss.resteasy.resteasy_jaxrs.i18n] (ServerService Thread Pool -- 61) RESTEASY002225: Deploying jakarta.ws.rs.core.Application: class org.wildfly.examples.GettingStartedApplication
+16:14:50,426 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 61) WFLYUT0021: Registered web context: '/' for server 'default-server'
+16:14:50,451 INFO  [org.jboss.as.server] (ServerService Thread Pool -- 35) WFLYSRV0010: Deployed "ROOT.war" (runtime-name : "ROOT.war")
+16:14:50,459 INFO  [org.jboss.as.server] (Controller Boot Thread) WFLYSRV0212: Resuming server
+16:14:50,460 INFO  [org.jboss.as] (Controller Boot Thread) WFLYSRV0060: Http management interface listening on http://0.0.0.0:9990/management
+16:14:50,460 INFO  [org.jboss.as] (Controller Boot Thread) WFLYSRV0054: Admin console is not enabled
+16:14:50,461 INFO  [org.jboss.as] (Controller Boot Thread) WFLYSRV0025: WildFly Full 31.0.0.Final (WildFly Core 23.0.1.Final) started in 1101ms - Started 280 of 378 services (149 services are lazy, passive or on-demand) - Server configuration file in use: standalone.xml
+----
+
+As you can, see the applications started, and it's ready to serve requests on port `8080`;
+
+NOTE: we explicitly mapped port `8080` in the container to port `8080` on the HOST (your PC); we also mapped port `9990` in the container to port `9990` on the HOST to expose the Liveness, Readiness and Startup Probes;
+
+=== Check the application
+
+Just like we did before, go to this URL:
+
+* link:http://127.0.0.1:8080/hello/pippo[http://127.0.0.1:8080/hello/pippo]
+
+either in your browser, or using a utility like `curl`:
+
+[source,bash]
+----
+$ curl http://127.0.0.1:8080/hello/pippo
+Hello 'pippo'.
+----
+
+The only difference here, is that the response is served by the application running inside our `{my-jaxrs-app-docker-image-name}:latest` Docker Image;
+
+=== Check Liveness, Readiness and Startup Probes
+
+Same as before, go to these URLs:
+
+* link:http://127.0.0.1:9990/health/live[http://127.0.0.1:9990/health/live]
+* link:http://127.0.0.1:9990/health/ready[http://127.0.0.1:9990/health/ready]
+* link:http://127.0.0.1:9990/health/started[http://127.0.0.1:9990/health/started]
+
+either in your browser, or using a utility like `curl`:
+
+[source,bash]
+----
+$ curl http://127.0.0.1:9990/health/live
+{"status":"UP","checks":[{"name":"empty-liveness-checks","status":"UP"}]}
+----
+[source,bash]
+----
+$ curl http://127.0.0.1:9990/health/ready
+{"status":"UP","checks":[{"name":"suspend-state","status":"UP","data":{"value":"RUNNING"}},{"name":"server-state","status":"UP","data":{"value":"running"}},{"name":"deployments-status","status":"UP","data":{"ROOT.war":"OK"}},{"name":"boot-errors","status":"UP"},{"name":"ready-deployment.ROOT.war","status":"UP"}]}
+----
+[source,bash]
+----
+$ curl http://127.0.0.1:9990/health/started
+{"status":"UP","checks":[{"name":"started-deployment.ROOT.war","status":"UP"}]}
+----
+
+== What's next?
```

Comment:
Add how to stop the image at this point? Otherwise you will moved to the next section with the image still running locally

Reply:
@yersan added it at the end!